static/app/utils/replays/playback/hooks/useReplayPlayWhenVisible.tsx

```diff
@@ -0,0 +1,24 @@
+import {useEffect} from 'react';
+
+import useReplayPlayerState, {
+  useReplayUserAction,
+} from 'sentry/utils/replays/playback/providers/useReplayPlayerState';
+
+export default function useReplayPlayWhenVisible() {
```

Comment:
Doesn't this pause when not visible?

Reply:
I don't think that rrweb pauses on it's own, we have code for that in replayContext.tsx

Reply:
I mean this hook, look at line 14

Reply:
oh, it's named backwards! derp took me long enough

Reply:
also it's not part of this diff anymore, because nothing plays for hydration errors. i'll note it over here which is sort of the OG branch: https://github.com/getsentry/sentry/pull/75175/files#r1747319889